Doc/c-api/apiabiversion.rst

```diff
@@ -60,7 +60,7 @@ See :ref:`stable` for a discussion of API and ABI stability across versions.
 
    Use this for numeric comparisons, e.g. ``#if PY_VERSION_HEX >= ...``.
 
-   This version is also available via the symbol :data:`Py_Version`.
+   This version is also available via the symbol :c:var:`Py_Version`.
```

Comment:
See #107091. It is discussed whether we should use `:c:var:` or `:c:data:`.

#107063 also fixes this line.